content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/quota_source.rb

```diff
@@ -1,16 +1,43 @@
+# frozen_string_literal: true
```

Comment:
Hi @GregP I don't see this line in the other methods. Does it resolve a rubocop error?

Reply:
@tinaafitz This line should be there, speeds up processing until we move to a more recent Ruby release.